features/registration.feature

```diff
@@ -20,3 +22,20 @@ Feature: Registration
   Scenario: passed token parameter without argument
     When I run `SUSEConnect --regcode --url http://localhost:3000`
     And the exit status should be 1
+
+  Scenario: passed invalid regcode
+    When I run `SUSEConnect --regcode foo --url http://localhost:3000`
```

Comment:
As discussed registration.feature is not used. Will discuss whether to remove it in the standup.